bears/c_languages/CSecurityBear.py

```diff
@@ -33,12 +33,5 @@ class CSecurityBear:
     CAN_DETECT = {'Security', 'Memory Leak', 'Code Simplification'}
 
```

Comment:
hey, two things:
1. can you encaption neverignore in double backticks in commit message?
2. can you explain why this is needed in the commit body? me, for example, i have no idea ;D